swarm/src/behaviour.rs

```diff
@@ -123,37 +116,133 @@ pub(crate) type THandlerOutEvent<THandler> =
 ///   }
 /// }
 /// ```
-pub trait NetworkBehaviour: 'static {
+pub trait NetworkBehaviour: Sized + 'static {
```

Comment:
How come this has to be explicitly `Sized` now?

Reply:
I don't remember, let me try remove it and see what breaks!

Reply:
Not needed!